apps/strapi-cms/src/index.ts

```diff
@@ -1 +1,74 @@
-export default {};
+// Disable all eslint rules that directly contradict how Strapi works
+/* eslint-disable functional/no-return-void, functional/no-expression-statements, functional/immutable-data, functional/no-try-statements */
```

Comment:
Consider adding this file to [ignorePatterns](https://github.com/pagopa/b2b-portals/blob/main/apps/strapi-cms/.eslintrc.json#L6)

Reply:
Got it. In this case consider adding an overrides (like is done [here](https://github.com/pagopa/b2b-portals/blob/2b9be52/packages/eslint-config/eslint-functional.js#L27-L43))